tools/rosetta/lib/internal/service/online.go

```diff
@@ -25,6 +30,15 @@ func NewOnlineNetwork(network *types.NetworkIdentifier, client crgtypes.Client)
 		return OnlineNetwork{}, err
 	}
 
+	// Get genesis hash from ENV. It should be set by an external script since is not possible to get
+	// using tendermint API
+	genesisHash := os.Getenv(genesisHashEnv)
+	if genesisHash == "" {
+		_ = fmt.Sprintf("[Warning]- Genesis hash env '%s' is not properly set!", genesisHashEnv)
```

Comment:
Did you mean to invoke fmt.Printf instead? Why this if we are ignoring the string? I see a "Warning" hence we should figure out whether to return an error or not.